src/main/java/homeworks/daniel/firstAssigment/inheritanceExercise/TestDevice.java

```diff
@@ -0,0 +1,15 @@
+package homeworks.daniel.firstAssigment.inheritanceExercise;
+
+public class TestDevice {
+    public static void main(String[] args) {
+    testDevice(new PC());
+    testDevice(new SmartPhone());
+    testDevice(new Tablet());
+
+    }
+    private static void testDevice(Devices myDevice){
```

Comment:
Even though this is just for testing purposes, always remember the blank spaces, JavaDocs and indentations.